src/main/java/com/amazon/corretto/hotpatch/patch/impl/set/PatchSetPatcher.java

```diff
@@ -0,0 +1,177 @@
+                            logger.log(String.valueOf(uce));
+                        }
+                    }
+                }
+            }
+
+            if (patchesApplied) {
+                logger.log("Vulnerable classes were not found. This agent will continue to run " +
+                        "and transform the target classes if they are loaded. Note that if you have shaded " +
+                        "or otherwise changed the package name for target classes, then this tool may not " +
+                        "find them.");
+            }
+        }
+
+        return Patcher.SUCCESS;
+    }
+
+    @Override
+    public int uninstall() {
+        instrumentation.removeTransformer(transformer);
+        // Retransform after we've removed the transformer to restore the initial class versions.
+        for (Class<?> c : instrumentation.getAllLoadedClasses()) {
+            String className = c.getName();
+            for (ClassTransformerHotPatch patch : getPatches()) {
+                if (patch.isTargetClass(className)) {
+                    logger.log("Un-Patching " + c + " (" + c.getClassLoader() + ") of " + patch.getName());
+                    try {
+                        instrumentation.retransformClasses(c);
+                    } catch (UnmodifiableClassException uce) {
+                        logger.log(String.valueOf(uce));
+                    }
+                }
+            }
+        }
+        logger.log("Uninstalled Patcher: " + getName() + ". Version: " + getVersion());
+        return SUCCESS;
+    }
+
+    /**
+     * A detailed description for this PatchSet that includes the description of the individual patches that will be
+     * applied.
+     * @return A detailed description for this PatchSet that includes the description of the individual patches that
+     *         will be applied.
+     */
+    @Override
+    public String getFullDescription() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(getName()).append("#").append(getVersion()).append(":").append(System.lineSeparator());
+        sb.append("  ").append(getShortDescription()).append(System.lineSeparator());
+        sb.append("  Applies the following patches:").append(System.lineSeparator());
+        for (ClassTransformerHotPatch patch : getPatches()) {
+           sb.append("    ").append(patch.getName()).append(": ").append(patch.getDescription())
+                   .append(System.lineSeparator());
+        }
+        return sb.toString();
+    }
+
+    /**
+     * Filters the result of getPatches based on the agent arguments. This is because besides what patches are defined
+     * in a patch set, some of them can be disabled via command line arguments ,
+     * @param args String with the arguments that was received by the agent,
+     * @return A list with all the patches that are enabled,
+     */
+    private List<ClassTransformerHotPatch> getEnabledPatches(final Map<String, String> args) {
```

Comment:
I added support for it in the agent because updating the agent code is going to be complicated (due to it's need of ideally being forward and backward compatible). However, the feature to disable individual patches in a patchset is something I want to add later, it is not yet ready

Reply:
Please remove everything for it until it's ready